js/views/modals/EditListing.js

```diff
@@ -208,7 +251,34 @@ export default class extends BaseModal {
 
         canvas.width = imgW;
         canvas.height = imgH;
-        canvas.getContext('2d').drawImage(newImage, 0, 0, imgW, imgH);
+
+        switch (orientation) {
```

Comment:
What is the purpose of this code? When I comment out this code and I rotate an image on my computer and then upload, the rotation is respected. What am i missing?

![image](https://cloud.githubusercontent.com/assets/794517/19084963/5744ac8c-8a1d-11e6-8392-d6e2e40285d4.png)

Also, this code seems quite complex, would [this](http://stackoverflow.com/a/27882870/632806) not be simpler?